recipes/mmcv/meta.yaml

```
@@ -0,0 +1,130 @@
+        - torchvision
+        - setuptools
+      run:
+        # intentionally skipping opencv and pytorch (following upstream vendor)
+        - addict
+        - packaging
+        - Pillow
+        - pyyaml
+        - yapf
+        - {{ pin_compatible('numpy') }}
+        - typing_extensions
+      run_constrained:
+        - pytorch =*={{ proc_type }}*
+    test:
+      requires:
+        - pytest
+        - requests
+        - scipy
+        - libopencv
+        - py-opencv
+        - opencv
+        - python-lmdb
+        - libiconv
+        - tifffile
+        - python-dateutil
+        - torchvision
+      imports:
+        - mmcv
+      source_files:
+        - tests
+      commands:
+        {% set failing_tests = ["test_imcrop",
+                                "test_tensor2imgs",
+                                "test_imwrite",
+                                "test_imfrombytes",
+                                "test_imread"] %}
+        {% set slow_tests = ["test_eval_hook",
+                             "test_start_param",
+                             "test_flat_cosine_runner_hook",
+                             "test_momentum_runner_hook",
+                             "test_step_runner_hook",
+                             "test_adjust_sharpness",
+                             "test_adjust_contrast",
+                             "test_adjust_brightness",
+                             "test_gradient_cumulative_optimizer_hook",
+                             "test_auto_contrast"] %}
+        {% set ignored_files = ["./tests/test_ops/test_onnx.py",
+                                "./tests/test_ops/test_tensorrt.py",
+                                "./tests/test_ops/test_tensorrt_preprocess.py",
+                                "./tests/test_utils/test_timer.py",
+                                "./tests/test_utils/test_progressbar.py"] %}
+        # ignored tests:
+        # test_onnx.py,test_tensorrt.py,test_tensorrt_preprocess.py - mccv currently compiled without onnx runtime
+        # test_geometric.py:test_imcrop - depending on what backend opencv uses it produces diff results (lipjpeg vs libturbojpeg)
+        # test_io.py - tries to import libturbojpeg that we don't install (conflicts with opencv that uses libjpeg)
+        # test_progressbar.py, test_timer.py - on OSX sleep(1) takes 1.1sec, so all related tests fail
+        # test_image_misc.py::test_tensor2imgs - need to investigate 2/75 mismatch
+        - python3 -m pytest ./tests/ -k "not ({{ (failing_tests + slow_tests)|join(' or ') }})" --ignore={{ ignored_files|join(' --ignore=') }}
+        - python3 -c "import mmcv; print(mmcv.__version__)"
+
+
+about:
+  home: https://github.com/open-mmlab/mmcv
+  summary: 'Foundational library for computer vision research'
+  description: |
+    MMCV is a foundational library for computer vision research and supports many research projects.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```

Why are you using the custom file? You should be able to pick it up from the source itself.
```suggestion
  license_file: LICENSE
```